install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1298,7 +1298,7 @@ $ oc project logging
 ifdef::openshift-enterprise[]
 ----
 $ oc apply -n openshift -f \
-    /usr/share/ansible/openshift-ansible/roles/openshift_hosted_templates/files/v1.4/enterprise/logging-deployer.yaml
+    /usr/share/ansible/openshift-ansible/roles/openshift_hosted_templates/files/v1.5/enterprise/logging-deployer.yaml
```

Comment:
@richm @jcantrill Anything else need to change in this "Upgrading the EFK Logging Stack" section other than the version bump here?